packages/minimal/scripts/setup/get-icons.js

```diff
@@ -0,0 +1,191 @@
+'use strict';
+const { exec, execSync } = require('child_process');
```

Comment:
i think this script needs to be fixed: https://github.com/open-rmf/rmf-web/issues/455